packages/framer-motion/src/projection/node/create-projection-node.ts

```diff
@@ -467,6 +467,15 @@ export function createProjectionNode<I>({
 
                             this.startAnimation(animationOptions)
                         } else {
+                            /**
+                             * If the layout hasn't changed and we have an animation that hasn't started yet,
+                             * finish it immediately. Otherwise it will be animating from a location
+                             * that was probably never commited to screen and look like a jumpy box.
+                             */
+                            if (!hasLayoutChanged && this.progress === 0) {
```

Comment:
If the layout hasn't changed, why did it start the animation? 🤔

Reply:
In a previous render where the layout did change. This is a new render that may have happened so soon that the old animation didn't even start to play.